app/pages/dashboard/template.njk

```diff
@@ -0,0 +1,33 @@
+{% extends 'includes/layout.njk' %}
+{% from 'components/back-link/macro.njk' import backLink %}
+{% from 'components/button/macro.njk' import button %}
+{% from './components/table.njk' import table %}
+
+{% block body %}
+  <div class="nhsuk-width-container" data-test-id="organisations">
+    <div data-test-id="go-back-link">
+      {{ backLink({
```

Comment:
Don't think we'll have anything to go back to just yet. Is this on the designs?

Reply:
not on the designs but consistent with the rest of the pages. Guessing will go back to pb homepage cos that's where the link is